service-front/module/Application/view/application/pages/national_insurance_number.twig

```diff
@@ -0,0 +1,53 @@
+<div class="govuk-width-container">
+    <main class="govuk-main-wrapper">
+        <h1 class="govuk-heading-xl">National Insurance Number</h1>
+        <form method="POST" target="">
+            <h3 class="govuk-heading-m">Name</h3>
+            <p class="govuk-body">
+                {{ details_data.Name }}
+            </p>
+            <h3 class="govuk-heading-m">Date of Birth</h3>
+            <p class="govuk-body">
+                {{ details_data.DOB }}
+            </p>
+            <h3 class="govuk-heading-m">National Insurance Number</h3>
+            <p class="govuk-body">
+                A NINO is made up of 2 letters, 6 numbers and a final letter, which is always A, B, C or D. Example QQ
+                12 34
+                56 A
+            </p>
+            <p class="govuk-body">
+
+                {% for message in form.get('nino').messages %}
+            <p id="event-name-error" class="govuk-error-message">
+                <span class="govuk-visually-hidden">Error:</span> {{ message }}
+            </p>
+            {% endfor %}
+            <input class="govuk-input govuk-!-width-one-third" id="nino" name="nino" type="text" autocomplete="nino"
+                   value="{{ form.get('nino').value }}"/>
+            </p>
```

Comment:
This should be using `.govuk-form-group` and `<label for="...">` like [the examples in the docs](https://design-system.service.gov.uk/components/text-input/).

When there's an error, the form group should also get `.govuk-from-group--error` and the input should get an aria-describedby pointing to the `<p>` that groups the error ([docs example](https://design-system.service.gov.uk/components/text-input/#error-messages))